api/envoy/extensions/transport_sockets/s2a/v3alpha/s2a.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package envoy.extensions.transport_sockets.s2a.v3alpha;
+
+import "udpa/annotations/migrate.proto";
```

Comment:
Do you need migrate.proto? IWYU please :)

Reply:
Followed [these](https://github.com/envoyproxy/envoy/blob/main/api/STYLE.md#adding-an-extension-configuration-to-the-api) guidelines and included migrate.proto. Will remove if it is OK to ignore that. I do see that currently, it is not used.

Reply:
Actually, went ahead and removed the import.